x/staking/abci.go

```diff
@@ -14,14 +14,20 @@ import (
 // BeginBlocker will persist the current header and validator set as a historical entry
 // and prune the oldest entry based on the HistoricalEntries parameter
 func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
-	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyBeginBlocker)
-
-	k.TrackHistoricalInfo(ctx)
+	if ctx.BlockHeight()%10 == 1 {
```

Comment:
what is this % 10 for?
Shouldn't we track telemetry every block?

Reply:
I think yes, we should track telemetry every block, will change